src/app/ranking/ranking-panel/ranking-panel.component.scss

```diff
@@ -139,12 +144,18 @@
 @media(min-width: $gtMobile) {
   .rank-location {
     @include altSmallCapsText(26px);
+    @include ellipsis-overflow();
     text-align: left;
     span {
       @include smallCapsText(14px);
     }
   }
 }
+@media(min-width: $gtTablet) {
```

Comment:
I'm guessing this is left over from testing?

Reply:
yep, very non-essential, will fix